pkg/crc/preflight/preflight_checks_linux.go

```diff
@@ -455,3 +460,31 @@ func fixLibvirtCrcNetworkActive() (bool, error) {
 	}
 	return true, nil
 }
+
+func checkCrcDnsmasqConfigFile() (bool, error) {
+	c := []byte(crcDnsmasqConfig)
+	_, err := os.Stat(crcDnsmasqConfigPath)
+	if err != nil {
+		return false, fmt.Errorf("File not found: %s: %s", crcDnsmasqConfigPath, err.Error())
+	}
+	config, err := ioutil.ReadFile(crcDnsmasqConfigPath)
+	if err != nil {
+		return false, fmt.Errorf("Error opening file: %s: %s", crcDnsmasqConfigPath, err.Error())
+	}
+	if !bytes.Equal(config, c) {
+		return false, fmt.Errorf("Config file contains changes: %s: %s", crcDnsmasqConfigPath)
```

Comment:
You forget to put the variable to second `%s` I think it should be `fmt.Errorf("Config file contains changes: %s: %s", crcDnsmasqConfigPath, string(config))

Reply:
Hmm, that was missed in the earlier PR? will change...

Reply:
Removed to dump the contents. Seems very unnecessary.